src/tests/functional/plugin/cpu/single_layer_tests/depth_to_space.cpp

```diff
@@ -214,30 +214,35 @@ INSTANTIATE_TEST_SUITE_P(smoke_CPUDepthToSpaceStaticBS3_5D, DepthToSpaceLayerCPU
 namespace dynamic_shapes {
 
 const std::vector<InputShape> inputShapes4D = {
-        {{-1, -1, -1 , -1},                                  // dynamic
-         {{2, 36, 1, 1}, {1, 36, 3, 1}, {1, 72, 1, 4}}},     // target
+    {{-1, -1, -1, -1},                                                                             // dynamic
+     {{2, 36, 1, 1}, {1, 36, 3, 1}, {2, 36, 1, 1}, {2, 36, 1, 1}, {1, 36, 3, 1}, {2, 36, 1, 1}}},  // target
```

Comment:
Though it is not so obvious, but placing two similar Target shapes one by one does not triggers the cache at all, since yet another optimization based on the fact that the shape has not been changed takes place. So I would say using the following Target shapes list would be enough:
{{2, 36, 1, 1}, {1, 36, 3, 1}, {2, 36, 1, 1}, {1, 36, 3, 1}, {1, 72, 1, 4}}}
Please check the other similar target shapes definitions
Another important thing is that the more target shapes we have, the longer the tests are passing because for each target shape case we run the reference implementation that may take a significant amount of time, but we have limited time range for the precommit tests. That means that we must take into account how much the new test cases extend the test time in comparison with the test coverage. For example, I looks like we can use one or two repeated target shapes per each definition to perform the cache smoke testing.